consumer/pdata/metric_test.go

```diff
@@ -39,18 +39,18 @@ func TestResourceMetricsWireCompatibility(t *testing.T) {
 	assert.NotNil(t, wire1)
 
 	// Unmarshal from the wire to OTLP Protobuf in goproto's representation.
-	var goprotoRM otlpmetricsgoproto.ResourceMetrics
-	err = goproto.Unmarshal(wire1, &goprotoRM)
+	var goprotoMessage emptypb.Empty
```

Comment:
I didn't know about `emptypb.Empty`. So this allows representing any message and fields are just stored as unknown fields?

Reply:
Yes. We can also if you want generate the protos with Google protobuf in the future. But the otel-proto are removed anyway so for the next update this is important to change

Reply:
No, this is good.